pkg/kubectl/cmd/testdata/edit/testcase-list-record/3.request

```diff
@@ -1,10 +1,5 @@
 {
 	"data": {
 		"new-data3": "newivalue"
-	},
```

Comment:
It used to be that one API actor could trick another into revealing their commands by setting an annotation that overrode local preference.  We fixed the bug.